packages/flutter/test/widgets/fade_transition_test.dart

```diff
@@ -3,14 +3,15 @@
 // found in the LICENSE file.
 
 import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
```

Comment:
I'm guessing this may have been auto-imported… would it be possible to remove it?

```suggestion
```